assets/dev/scss/frontend/widgets/rating.scss

```diff
@@ -0,0 +1,48 @@
+.e-rating {
+	--e-rating-gap: 0px;
+	--e-rating-star-font-size: 16px;
+	--e-rating-star-color: #{$gray-mouse};
+	--e-rating-star-color-value: #{$orange};
+	--e-rating-star-marked-width: 0%;
+
+	&-wrapper {
+		display: flex;
+		gap: var(--e-rating-gap);
+		flex-direction: row;
+		width: fit-content;
+	}
+
+	.e-star {
+		position: relative;
+
+		&-wrapper {
+
+			&.e-star-marked {
+				--e-rating-star-color: var(--e-rating-star-color-value);
```

Comment:
This is the color of the marked star.
I opted to redefine the CSS variable instead of having to add multiple lines to style the icon color and the SVG with the fill property. This method is quicker, but it could be too cryptic. What do you think?